<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PredictDialog</class>
 <widget class="QDialog" name="PredictDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>legnth</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>output1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>output2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>output3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>InputPosition</string>
       </property>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>OutputPosition</string>
       </property>
       <widget class="QCheckBox" name="checkBox_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PredictDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PredictDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
